test/parallel/test-net-better-error-messages-port-hostname.js

```diff
@@ -3,12 +3,12 @@ var common = require('../common');
 var net = require('net');
 var assert = require('assert');
 
-var c = net.createConnection(common.PORT, '...');
+var c = net.createConnection(common.PORT, '***');
```

Comment:
What's the reason for changing `...` to `***`? (Sorry if it's a naive question.)